### ISSUE TITLE
cannot import name '_mount_points_from_df_output' from 'trashcli.list_mount_points'

### ISSUE THREAD

**Issue by rycee** [andreafrancia/trash-cli#177]
Trying to update the [trash-cli package in Nixpkgs](https://github.com/NixOS/nixpkgs/blob/da6239dab3d712aa335fd0928c4e64f9599fef73/pkgs/tools/misc/trash-cli/default.nix#LC1) I got the test error

```
ERROR: Failure: ImportError (cannot import name '_mount_points_from_df_output' from 'trashcli.list_mount_points' (/build/source/trashcli/list_mount_points.py))
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/nix/store/0l7ns9sg0ykj1cvw5nqca2jbpn7cb6kc-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/failure.py", line 39, in runTest
    raise self.exc_val.with_traceback(self.tb)
  File "/nix/store/0l7ns9sg0ykj1cvw5nqca2jbpn7cb6kc-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/loader.py", line 416, in loadTestsFromName
    module = self.importer.importFromPath(
  File "/nix/store/0l7ns9sg0ykj1cvw5nqca2jbpn7cb6kc-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/nix/store/0l7ns9sg0ykj1cvw5nqca2jbpn7cb6kc-python3.8-nose-1.3.7/lib/python3.8/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/nix/store/5aq2ix4vmk8bxs20ddqi48vc4bnk8zr5-python3-3.8.6/lib/python3.8/imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "/nix/store/5aq2ix4vmk8bxs20ddqi48vc4bnk8zr5-python3-3.8.6/lib/python3.8/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 702, in _load
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/build/source/unit_tests/test_list_mount_points.py", line 5, in <module>
    from trashcli.list_mount_points import _mount_points_from_df_output
ImportError: cannot import name '_mount_points_from_df_output' from 'trashcli.list_mount_points' (/build/source/trashcli/list_mount_points.py)
```

It seems some test update was missed after the switch to psutil?

**Comment by andreafrancia** [comment]
Thank you.
I solved the problem with a new release the 0.20.11.23

**Comment by rycee** [comment]
Yes, that solves the issue. Thanks for the fix!
